packages/plan-form-core/src/formHelpers/activitiesLookup.ts

```diff
@@ -85,7 +81,6 @@ import {
   COMPLETE_FIX_PROBLEM_TASK,
   COMPLETE_FLAG_PROBLEM_TITLE,
   COMPLETE_FLAG_PROBLEM_DESCRIPTION,
-  FIX_PRODUCT_PROBLEM_ACTIVITY,
```

Comment:
nipick: Should we have this and the others not been used also removed on the language file?